lib/stackprof.rb

```diff
@@ -2,6 +2,7 @@
   require "stackprof/truffleruby"
 else
   require "stackprof/stackprof"
+  require "stackprof/tags"
```

Comment:
When using this on a project on `TruffleRuby` code using `StackProf::Tag` will fail (const not being loaded). If I'm not mistaken we don't need this for TruffleRuby, maybe we could stub it with a noop class and load that in that case.

Reply:
yeah i think we'll probably need a stub, as the usage of the StackProf::Tag helpers is intended to be throughout the codebase, so at least having the receivers available should prevent runtime errors, and not need to have every single call behind a an `if truffleruby` guard